### PR TITLE
Remove 2000Q content

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Very minor updates.
Note that the latest version seems to add ``attachments`` metadata: I opened in the IDE and it doesn't cause a problem but I can manually delete if anyone is worried.

### CODE REVIEW

File: 01-feature-selection.ipynb [hunk 1]
```diff
@@ -746,7 +747,7 @@
...
     "\n",
-    "Helper function `qpu_working_graph` below creates a [*dwave-networkx*](https://docs.ocean.dwavesys.com/en/stable/docs_dnx/sdk_index.html) graph that represents the *working graph* of the QPU selected by [DWaveSampler](https://docs.ocean.dwavesys.com/en/stable/docs_system/reference/samplers.html), a Pegasus or Chimera graph with the same sets of nodes (qubits) and edges (couplers) as the QPU. Ocean software's [*minorminer*](https://docs.ocean.dwavesys.com/en/stable/docs_minorminer/sdk_index.html) finds an embedding for the required clique size in the working graph. "
+    "Helper function `qpu_working_graph` below creates a [*dwave-networkx*](https://docs.ocean.dwavesys.com/en/stable/docs_dnx/sdk_index.html) graph that represents the *working graph* of the QPU selected by [DWaveSampler](https://docs.ocean.dwavesys.com/en/stable/docs_system/reference/samplers.html), such as a Pegasus graph with the same sets of nodes (qubits) and edges (couplers) as the QPU. Ocean software's [*minorminer*](https://docs.ocean.dwavesys.com/en/stable/docs_minorminer/sdk_index.html) finds an embedding for the required clique size in the working graph. "
```
[the remark this answers is not in the document]
Comment (JoelPasvolsky) [reply 01-feature-selection.ipynb:750]: Good catch! [This PR](https://github.com/dwave-examples/feature-selection-notebook/commit/deb63b79d7cb967ed790385ba5d8d5cdadacc79e) replaced my earlier ``qpu_working_graph`` helper function (under the local `helpers` folder) with the then-new ``DWaveSampler.to_networkx_graph``. The intro text was not updated to reflect that

File: 01-feature-selection.ipynb [hunk 2]
```diff
@@ -769,10 +770,11 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This problem is small enough to be solved in its entirety on a D-Wave 2000Q QPU. For datasets with higher numbers of features, D-Wave Ocean's [dwave-hybrid](https://docs.ocean.dwavesys.com/en/stable/docs_hybrid/sdk_index.html) tool can be used to break the BQM into smaller pieces for serial submission to the QPU and/or parallel solution on classical resources. Here, an out-of-the-box hybrid sampler, [Kerberos](https://docs.ocean.dwavesys.com/en/stable/docs_hybrid/reference/reference.html) is used.    "
+    "This problem is small enough to be solved in its entirety on an Advantage QPU. For datasets with higher numbers of features, D-Wave Ocean's [dwave-hybrid](https://docs.ocean.dwavesys.com/en/stable/docs_hybrid/sdk_index.html) tool can be used to break the BQM into smaller pieces for serial submission to the QPU and/or parallel solution on classical resources. Here, an out-of-the-box hybrid sampler, [Kerberos](https://docs.ocean.dwavesys.com/en/stable/docs_hybrid/reference/reference.html) is used.    "
```
[the remark this answers is not in the document]
Comment (JoelPasvolsky) [reply 01-feature-selection.ipynb:777]: Removed, thank you

File: 01-feature-selection.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (JoelPasvolsky) [reply 01-feature-selection.ipynb]: Updated

**Review by mdecandia — APPROVED** [review]
Looks good

**Review by randomir — APPROVED** [review]

**Pull request merged**